.github/workflows/deploy_production.yml

```diff
@@ -10,18 +10,9 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
-    name: Build production
-    uses: zooniverse/ci-cd/.github/workflows/npm_build.yaml@main
-    with:
-      commit_id: ${{ github.sha }}
-      node_version: '16.x'
-      output: 'build'
-      script: 'build'
   deploy_production:
     name: Deploy production
     uses: zooniverse/ci-cd/.github/workflows/deploy_static.yaml@main
-    needs: build
```

Comment:
On second thoughts, I think the deploy downloads the HTML and CSS files from the build step, so this should stay and we should have a build step that uploads the HTML and CSS.